client/src/components/NavBar/NavBar.tsx

```diff
@@ -89,23 +92,51 @@ export function Navbar(props: any) {
 
   return (
     <NavbarStyled>
-      <details>
-        <summary>menu</summary>
+      <ButtonWrapper>
+        <MenuButton
+          className="menu-button"
+          onClick={() => {
+            setIsNavExpanded(!isNavExpanded);
+          }}
+        ></MenuButton>
+      </ButtonWrapper>
 
-        <Link to="/">Home</Link>
-        <br />
-        <Link to="advanced-search">Advanced Search</Link>
-        <br />
-        <Link to="review">Review</Link>
-        <Link to="reviews">Reviews</Link>
-        <br />
-        <Link to="login">Login</Link>
-        <Link to="sign-up">Sign Up</Link>
-        <StyledButton onClick={() => testAPICalls()}>Test API</StyledButton>
-        <Link to={"/single-result/ChIJn58N1B9gUocRpAXOXPbFcOo"}>
-          Single Result
-        </Link>
-      </details>
+      <NavMenu>
+        <div
+          onClick={() => {
+            setIsNavExpanded(!isNavExpanded);
+          }}
+          className={isNavExpanded ? "expanded" : ""}
+        >
+          <ul>
+            <li>
+              {" "}
```

Comment:
What are the purpose of these strings? If its for spacing, then we should do this in css, not jsx.

Reply:
It's something VSCode does every once in a while. I deleted them.